spec/acceptance/member_spec.rb

```diff
@@ -12,4 +12,54 @@
       expect(output).to match(/8332-fcdaecb13e34 | John Doe | Administrator/)
     end
   end
+
+  describe "add" do
+    it "adds a new member to a space" do
+      command = %W(
+        member add
+        --space-id #{invitation.space_id}
+        --user-id #{invitation.id1}:#{invitation.role}
+        --email #{invitation.email1}:0 #{invitation.email2}:1
+        --message #{invitation.message}
+      )
+
+      stub_ribose_space_invitation_mass_create(
+        invitation.space_id, build_attr_in_stub_format(invitation)
+      )
+
+      output = capture_stdout { Ribose::CLI.start(command) }
+
+      expect(output).to match(/Invitation has been sent successfully!/)
+    end
+  end
+
+  def invitation
+    @invitation ||= OpenStruct.new(
+      id1: "123456",
+      id2: "567890",
+      role: "123456",
+      space_id: "123456789",
+      email1: "invitee-one@example.com",
+      email2: "invitee-two@example.com",
+      message: "Your details invitation message",
```

Comment:
Actually, I think we don't really need "details" here.  So, "Your invitation message" should suffice.